lighthouse-core/lib/csp-evaluator.js

```diff
@@ -76,6 +75,18 @@ const UIStrings = {
   /** Message shown when a CSP uses the deprecated disown-opener directive. Shown in a table with a list of other CSP vulnerabilities and suggestions. "CSP" stands for "Content Security Policy". "disown-opener", "CSP3" and "Cross-Origin-Opener-Policy" do not need to be translated. */
   deprecatedDisownOpener: 'disown-opener is deprecated since CSP3. ' +
     'Please, use the Cross-Origin-Opener-Policy header instead.',
+  /**
+   * @description Message shown when a CSP wildcard allows unsafe scripts to be run in the page. Shown in a table with a list of other CSP vulnerabilities and suggestions. "CSP" stands for "Content Security Policy".
+   *  @example {https:} keyword
+   */
+  wildcard: 'Avoid using wildcards ({keyword}) in this directive. ' +
+    'It can allow scripts to be sourced from an unsafe domain.',
+  /**
+   * @description Message shown when a CSP URL scheme allows unsafe scripts to be run in the page. Shown in a table with a list of other CSP vulnerabilities and suggestions. "CSP" stands for "Content Security Policy".
+   *  @example {https:} keyword
+   */
+  plainUrlScheme: 'Avoid using plain URL schemes ({keyword}) in this directive. ' +
+    'It can allow scripts to be sourced from an unsafe domain.',
```

Comment:
same nit, also the `can` seems *too* cautious, this just opens it up to the entire internet right?
```suggestion
    'Plain URL schemes allow scripts to be sourced from an unsafe domain.',
```